test/LinalgTransform/invalid.mlir

```diff
@@ -12,14 +12,6 @@ linalg_transform.sequence {
 
 // -----
 
-linalg_transform.sequence {
```

Comment:
isn't there some positive mlir test to update that exercises padding chaining after tile?